examples/v1beta1/taskruns/optional-resources.yaml

```diff
@@ -1,133 +0,0 @@
-apiVersion: tekton.dev/v1beta1
-kind: Task
-metadata:
-  name: task-check-optional-resources
```

Comment:
optional resources have not been removed yet, but since all resources are going away it's not worth migrating this test